tests/patmat/i4880a.check

```diff
@@ -0,0 +1 @@
+14: Match case Unreachable
```

Comment:
This is new, and related to #4880, but seems correct: https://github.com/lampepfl/dotty/blob/master/tests/patmat/i4880a.scala#L14